control-plane/pkg/reconciler/broker/namespaced_broker.go

```diff
@@ -395,3 +462,35 @@ func appendOwnerRef(refs []metav1.OwnerReference, broker *eventing.Broker) ([]me
 
 	return append(refs, newRef), true
 }
+
+func filterMetadata() mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		u.SetLabels(filterMetadataMap(u.GetLabels()))
+		u.SetAnnotations(filterMetadataMap(u.GetAnnotations()))
+		return nil
+	}
+}
+
+func setLabel() mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		labels := u.GetLabels()
+		labels[kafka.NamespacedBrokerDataplaneLabelKey] = kafka.NamespacedBrokerDataplaneLabelValue
+		u.SetLabels(labels)
+		return nil
+	}
+}
+
+func filterMetadataMap(metadata map[string]string) map[string]string {
+	r := make(map[string]string, len(metadata))
+	for k, v := range metadata {
+		if strings.Contains(k, "knative") || k == "app" {
+			r[k] = v
+		}
+	}
+	return r
+}
```

Comment:
is this really needed?

Reply:
Don't know. I didn't add it.

Didn't you add that part?

Reply:
I think I added it but I don't think we need it

Reply:
I thought there were some issues and I trusted your code 😆

Reply:
@pierDipi What was your original motivation for this code?